pkg/api/redistribution.go

```diff
@@ -22,6 +22,7 @@ type redistributionStatusResponse struct {
 	LastWonRound       uint64         `json:"lastWonRound"`
 	LastPlayedRound    uint64         `json:"lastPlayedRound"`
 	LastFrozenRound    uint64         `json:"lastFrozenRound"`
+	LastSelectedRound  uint64         `json:"lastSelectedRound"`
```

Comment:
This should be also reflected in the OpenAPI spec.

Reply:
@quinton11 hey 🌞

Reply:
@vladopajic  I reverted the changes made to the openapi spec to prevent triggering a new upload. Would open another pr from a clone with the changes and link to this pr, what do you think?

Reply:
Thanks for being cautious. It might have more sense to add that change in this PR.

Reply:
If I add the change to this PR the OpenApi job will fail at the credential check. Or I'm not getting your meaning?

Reply:
It will, but just ignore it. Bee team members can merge PRs avoiding CI checks.